data/json/items/tool_armor.json

```
@@ -619,6 +619,34 @@
     "warmth": 60,
     "magazine_well": "250 ml"
   },
+  {
+    "id": "thermal_super",
+    "copy-from": "thermal_outfit",
+    "name": { "str": "thermal electric survival outfit" },
+    "description": "This skintight suit covers you from head to toe and is designed to keep you at an ideal temperature using a number of heatsinks and heating coils.  It also includes some handy features such as a built-in thermometer.  Use it to turn it on.",
+    "flags": [ "VARSIZE", "SKINTIGHT", "TRADER_AVOID", "ONLY_ONE", "ALARMCLOCK", "THERMOMETER" ],
```

```suggestion
    "flags": [ "VARSIZE", "SKINTIGHT", "TRADER_AVOID", "ONLY_ONE", "THERMOMETER" ],
```

Has `ALARMCLOCK` flag but recipe only adds a thermometer.

---

I was debating adding a watch to the recipe and making it a clock, but I decided not to- and apparently forgot to remove the flag.